data/templates/install-coreos.sh

```diff
@@ -2,4 +2,5 @@
 set -e
 curl -O http://<%=server%>:<%=port%>/api/current/templates/pxe-cloud-config.yml
 sudo coreos-install -d <%=installDisk%> -c pxe-cloud-config.yml -b <%=repo%>
+wget --spider http://<%=server%>:<%=port%>/api/current/templates/<%=completionUri%>
```

Comment:
Thanks for the feedback!

I used `--spider` because that's what I saw here: https://github.com/RackHD/on-http/blob/master/data/templates/centos-ks#L216

Truth be told, there are lots of different ways I saw that being done. some with wget, some with curl, and all different variations of options.  However, `--spider` seemed like not a bad one, and I don't think it would download a directory. The help says `       --spider                    don't download anything`.  It just sees if the file is there.  But it sure might crawl a directory. I don't see any examples of completionUri being a directory in the code though.

I'm testing with "CoreOS stable (1068.8.0)", and it has wget. not sure about older versions.